cdap-runtime-spi/pom.xml

```diff
@@ -39,6 +39,12 @@
       <groupId>io.cdap.twill</groupId>
       <artifactId>twill-api</artifactId>
     </dependency>
+      <dependency>
+          <groupId>io.cdap.cdap</groupId>
+          <artifactId>cdap-error-api</artifactId>
+          <version>${project.version}</version>
+          <scope>compile</scope>
```

Comment:
don't need to specify scope when it's compile, unless it's overriding some other scope (which this isn't)

Reply:
Done.